indigo_content_api/v3/serializers.py

```diff
@@ -0,0 +1,32 @@
+from indigo_content_api.v2.serializers import PublishedDocumentSerializer as PublishedDocumentSerializerV2
+
+
+class PublishedDocumentSerializerV3(PublishedDocumentSerializerV2):
+    """ Serializer for published documents.
+    Inherits most fields from the v2 published document serializer.
+    - Excludes detailed commencement information.
+    - Excludes taxonomies.
+    """
+    # TODO: is this right, or just change Meta.fields?
+    commencements = None
```

Comment:
i think you have to do this, otherwise it  will complain that there is a field declared but not included in `fields`